shopsdk/setup.py

```diff
@@ -0,0 +1,149 @@
+		m("\n")
+		m("so do not go saying I stole credit, as I take no credit and we will...\n")
+		m("Just virtually ignore you if you say it.\n")
+		import acwc24
+		ad=["anniversary_cake",
+			"blue_Pikmin_hat",
+			"bus_model",
+			"Chihuahua_model",
+			"dachshund_model",
+			"dalmatian_model",
+			"Dolphin_model",
+			"Eiffel_tower",
+			"fedora_chair",
+			"flamenco_hat",
+			"GameCube_dresser",
+			"Gracie_dresser",
+			"green_headgear",
+			"guard_s_helmet",
+			"hopscotch_floor",
+			"Kapp_n_model",
+			"Labrador_model",
+			"ladder_shades",
+			"Nintendo_DS_Lite",
+			"Nintendo_DSi_B",
+			"Nintendo_DSi_W",
+			"Pave_clock",
+			"red_Pikmin_hat",
+			"shopping_cart",
+			"tam_o_shanter",
+			"Wii_locker",
+			"wildflower_floor",
+			"yellow_Pikmin_hat"]
+		aa={}
+		aa[1]=["snowman_head", "snowman_vanity"]
+		aa[2]=["Cupid_bench"]
+		aa[3]=["egg_TV", "shamrock_hat"]
+		aa[4]=["egg_TV"]
+		aa[5]=[]
+		aa[6]=["banana_split_hat", "hot_dog_hat", "sand_castle"]
+		aa[7]=["banana_split_hat", "hot_dog_hat", "sand_castle"]
+		aa[8]=["banana_split_hat", "hot_dog_hat", "sand_castle"]
+		aa[9]=["pile_of_leaves"]
+		aa[10]=["pile_of_leaves"]
+		aa[11]=["pile_of_leaves"]
+		aa[12]=["snowman_head", "snowman_vanity", "Jingle_TV", "festive_wreath"]
+		def picker():
+			month=digitalcontentsender.getcurrentmonth()
+			ad_all=ad + aa[month]
+			choice=j.choices(ad_all, weights=[1] * len(ad) + [2] * len(aa[month]), k=1)[0]
+			return choice
+		if os.path.exists("dlc.dat"):
+			dlc_list=i.load(open("dlc.dat", "rb"))
+			choice=list(dlc_list.values())[-1]
+			while choice in list(dlc_list.values())[:len(ad)]:
+				choice=picker()
+			dlc_id=list(dlc_list.keys())[-1] + 1
+		else:
+			dlc_list={}
+			choice=picker()
+			dlc_id=1
+		m("The next DLC item will be: " + choice + "!")
+		dlc_list[dlc_id]=choice
+		i.dump(dlc_list, open("dlc.dat", "wb"))
+		a.create(choice, False, 8192 + dlc_id)
+		region2={}
+		region2["E"]="us"
+		region2["P"]="eu"
+		region2["J"]="jp"
+		region2["K"]="kr"
+		for region in ["E", "P", "J", "K"]:
+			l.call(["mv", "build/" + choice + "_" + region + ".arc.wc24", "/var/www/wapp.wii.com/nwcs/public_html/ruu/rvforestdl_" + region2[region] + ".enc"])
+		dlc_message="We are now distributing this item:\n\n" + choice + "\n\nEnjoy!"
+		data={"username": "Animal Crossing DLC Bot", "content": dlc_message,
+			"avatar_url": "http://rc24.xyz/images/logo-small.png", "attachments": [
+				{"fallback": dlc_message, "color": "#549537", "author_name": "RiiConnect24 Animal Crossing DLC Script",
+					"author_icon": "https://rc24.xyz/images/ajs/animalcrossing/pete.png",
+					"text": dlc_message, "title": "Update!",
+					"fields": [{"title": "Script", "value": "Animal Crossing Wii", "short": "false"}],
+					"footer": "RiiConnect24 Script",
+					"footer_icon": "https://rc24.xyz/images/logo-small.png",
+					"ts": int(b.timegm(digitalcontentsender.getutc().timetuple()))}]}
+		aj=open(discord.dat, 'rb')
```

Comment:
if this is going to be included, could it please be in its own repository outside of Wii no Ma?